torch/_subclasses/meta_utils.py

```diff
@@ -523,8 +523,32 @@ def __call__(
                 # instrumentation will see the meta conversions and the
                 # tests all break so we just exclude this.  In any case
                 # the to conversion isn't really right anyhow.
+                (
```

Comment:
This code probably shouldn't run for all of the cases listed above? (quantized/nested/sparse, etc)